DataCollection/src/Domain/Managers/BirdSpeciesManager.cs

```diff
@@ -33,9 +35,12 @@ public BirdSpeciesDomainManager(IBirdSpeciesPersistence birdSpeciesPersistence)
 
         public IEnumerable<BirdSpecies> GetBirdSpeciesBySurveyTypeId(int surveyTypeId)
         {
-            var retVal = BirdSpeciesPersistence.GetBirdSpeciesBySurveyTypeId(surveyTypeId);
-            return retVal;
-        }
+            var ctx = DataCollectionContext.Create();
+            var returnVal = new List<BirdSpecies>();
+            var surveyType = ctx.SurveyTypes.Include("BirdSpecies").FirstOrDefault(surveyItem => surveyItem.Id == surveyTypeId);
+            returnVal = surveyType?.BirdSpecies?.ToList();
```

Comment:
Does that compile? :confused: I didn't realize that we can use C# 6.0 with .NET 4.5.2 when compiling in Visual Studio 2015. I wonder what will happen when I put that on server? I just tried another 6.0 feature and it compiled and ran just fine locally. Worried about the impact on the remote server. That would be awesome though. I suppose I hadn't read enough about Roslyn to realize that it might enable this. Here I was thinking that we only get C# 6.0 with .NET 4.6, and I know that my server doesn't support 4.6...